services/provider/interfaces/interfaces.go

```diff
@@ -16,6 +18,8 @@ type StorageClientOnboarding interface {
 	GetOnboardingTicket() string
 	GetConsumerName() string
 	GetClientOperatorVersion() string
+	GetClusterId() string
+	GetStorageClientNamespacedName() string
```

Comment:
As of a similar reason to a previous comment, I would suggest the following rename:
 ```suggestion
	     GetNamespacedName() string
```

Also, aren't we missing the compliment `SetNamespacedName`?